Update SSH private key generation in the latest OKD installation documentation (#29044)
openshift/openshift-docs: modules/ssh-agent-using.adoc

====
endif::[]

.Procedure

. If you do not have an SSH key that is configured for password-less authentication
on your computer, create one.
For example, on a computer that uses a Linux operating system, run the
following command:
+
ifndef::openshift-origin[]
----
$ ssh-keygen -t rsa -b 4096 -N '' \
    -f <path>/<file_name> <1>
----
endif::openshift-origin[]
ifdef::openshift-origin[]
----
$ ssh-keygen -t ed25519 -N '' \
    -f <path>/<file_name> <1>
----
endif::openshift-origin[]

> <1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the SSH key.
> Do not specify an existing SSH key, as it will be overwritten.

Reviewer: It took me a couple seconds to digest this advice. Have you seen user run this and accidentally delete ssh keys?

```suggestion
<1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the SSH key to create.
```
